sql/rowexec/ddl_iters.go

```diff
@@ -1849,6 +1855,7 @@ func (b *BaseBuilder) executeCreateCheck(ctx *sql.Context, c *plan.CreateCheck)
 	if err != nil {
 		return err
 	}
+	rowIter = withSafepointPeriodicallyIter(rowIter)
```

Comment:
I don't know if this makes sense here, creating a check does do a table scan but it's read-only